src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -44,5 +50,8 @@
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <Target Condition="'$(TargetsWindows)' == 'true'" Name="BuildNetfxRemoteInvoker" AfterTargets="BeforeBuild">
+    <MSBuild Projects="RemoteInvokeNetfx\RemoteInvokeNetfx.csproj" Properties="Configuration=Debug; Platform=$(Platform); OutputPath=$(OutputPath); IntermediateOutputPath=$(IntermediateOutputPath)" />
```

Comment:
the targeting pack for netfx won't be fully complete in uapaot vertical builds, so you might find that to be a problem. One less hacky approach to do this might be to make the change of this new remoteInvoke in buildtools and have that restored here. Or even better, shouldn't we just fix the RemoteInvoke that we already use for the rest of the tests?